tests/main/interfaces-cups/task.yaml

```diff
@@ -0,0 +1,66 @@
+summary: Ensure that the cups interfaces work with app providers
+
```

Comment:
oh we probably need a `systems: [ubuntu-*]` or something for the platforms that have strict confinement, the test as written fails on systems without strict confinement:

```
2020-07-02T18:40:21.1169826Z ##[error]2020-07-02 18:40:21 Error executing google:fedora-32-64:tests/main/interfaces-cups (jul021808-861954) : 
2020-07-02T18:40:21.1179219Z -----
2020-07-02T18:40:21.1180102Z + test-snapd-provider.sh -c 'echo slot > /run/cups/cups.sock'
2020-07-02T18:40:21.1180405Z WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement
2020-07-02T18:40:21.1181129Z + test-snapd-provider.sh -c 'echo slot > /run/cups/other'
2020-07-02T18:40:21.1181408Z WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement
2020-07-02T18:40:21.1182151Z + not test-snapd-consumer.cups-control -c 'head /run/cups/cups.sock'
2020-07-02T18:40:21.1183071Z WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement
2020-07-02T18:40:21.1183344Z slot
2020-07-02T18:40:21.1183992Z -----
```